test/jdk/javax/swing/JInternalFrame/6725409/bug6725409.java

```diff
@@ -147,14 +137,13 @@ private void checkResult() {
     private static void sync() {
         robot.waitForIdle();
     }
-    private static ExtendedRobot createRobot() {
+
+    private static Robot createRobot() {
         try {
-             ExtendedRobot robot = new ExtendedRobot();
-             return robot;
-         }catch(Exception ex) {
-             ex.printStackTrace();
-             throw new Error("Unexpected Failure");
-         }
+            return new Robot();
+        } catch (Exception ex) {
+            throw new Error("Can't create Robot", ex);
+        }
```

Comment:
Same with com.sun.java.swing.plaf.windows.WindowsInternalFrameTitlePane , fully qualified name can be replaced.